lib/pages/chat/chat.dart

```diff
@@ -377,6 +378,13 @@ class ChatController extends State<ChatPageWithRoom>
     setState(() {});
   }
 
+  // #Pangea
+  void closeEmojiPicker() {
```

Comment:
I think this can be removed and reference to it replace by the function hideEmojiPicker()